frontend/src/patientApp/timeOfTest/constants.ts

```diff
@@ -134,3 +135,38 @@ export const syphilisSymptomDefinitions: SymptomDefinitionMap[] =
 export const SYMPTOM_SUBQUESTION_ERROR =
   "This question is required if the patient has symptoms.";
 export const ONSET_DATE_LABEL = "When did the patient's symptoms start?";
+
+export type AllSymptomsCode = keyof RespiratorySymptoms &
+  keyof SyphilisSymptoms;
+export type AllSymptomsName = RespiratorySymptomName & SyphilisSymptomName;
+export const allSymptomsMap = {
```

Comment:
⚠️ I tried to programmatically generate this map by combing the disease specific maps in code. However, since those maps are exported as constants they are defined as readonly. Because of this, I could not find a way programmatically generate an all symptoms map and instead just had to hard code this one.

Reply:
If someone does know a way to combine two readonly maps in code please lmk 😅

Reply:
I think you can just do 

```
export const allSymptomsMap = {
  ...syphilisSymptomsMap,
  ...respiratorySymptomsMap,
};
```

Reply:
Wow idk how I missed that 🤦 -- updated